src/app/extensions/quoting/shared/quote-interactions/quote-interactions.component.html

```diff
@@ -6,8 +6,13 @@
         data-testing-id="submit-quote-request"
         type="submit"
         class="btn btn-primary float-right"
-        [disabled]="quote.items.length === 0"
-        (click)="submit()"
+        [ngClass]="{ disabled: quote.items.length === 0 }"
+        [attr.aria-disabled]="quote.items.length === 0"
+        [attr.aria-label]="
+          (quote.items.length !== 0 ? 'quote.edit.button.submit.label' : 'quote.edit.button.submit.label.disabled')
+            | translate
+        "
+        (click)="quote.items.length !== 0 && submit()"
```

Comment:
the length check should be moved to the submit function - logic should be done in the component.ts file whenever possible